packages/reactstrap-validation-select/types/AvSelect.d.ts

```diff
@@ -5,6 +5,7 @@ export interface AvSelectProps extends AvInput {
   loadOptions?: Function;
   raw?: boolean;
   autofill?: boolean | object;
+  ariaMessage?: Object;
```

Comment:
ariaFeedback

Reply:
@jordan-a-young along the same lines as PropTypes, should we do an interface for ariaFeedback?

Reply:
If we know the shape then I think we should define it here